app/Controllers/CtrlApiFiles.php

```diff
@@ -83,6 +83,26 @@ public function processTemporalFile()
         }
     }
 
+    public function processTemporalFileByChunks()
+    {
+        try {
+            $key        = $this->request->getGet('patch');
+            $fileName   = $this->request->header('Upload-Name')->getValue();
+            $fileLength = $this->request->header('Upload-Length')->getValue();
+            $fileData   = $this->request->getBody();
+            $offset     = $this->request->header('Upload-Offset')->getValue();
+
+            $folder  = FILES_TEMP_DIRECTORY . $key;
+            $fileTmp = "{$folder}/{$fileName}";
+
+            $temporalFileLength = FileManager::mergeChunckFiles($fileTmp, $fileData, $offset);
+
+            return $this->response->setStatusCode(204);
+        } catch (Throwable $th) {
+            return $this->response->setStatusCode(500, json_encode('Ha ocurrido un error mientras se cargaba el archivo'));
```

Comment:
```suggestion
             return $this->response->setStatusCode(Response::HTTP_NO_CONTENT);
        } catch (Throwable $th) {
            return $this->response->setStatusCode(Response::HTTP_INTERNAL_SERVER_ERROR , json_encode('Ha ocurrido un error mientras se cargaba el archivo'));
        }
```
Suggestion for better code readability and maintenance
@Mauriciocr207, I recommend using constant variables for HTTP status codes throughout this file. 
This approach not only enhances code readability but also makes the reasons behind each status code more clear.
Ensure to import the `use CodeIgniter\HTTP\Response;` class.